src/Views/Form/Themes/Sequoia/optionConfig.php

```diff
@@ -1,26 +1,40 @@
 <?php
+global $post;
 return [
 	'introduction'        => [
 		'name'   => __( 'Introduction', 'give' ),
 		'desc'   => __( 'Step description will show up here if any', 'give' ),
 		'fields' => [
+			[
+				'name'    => __( 'Include Introduction', 'give' ),
+				'desc'    => sprintf( __( 'Should this form include an introduction section?', 'give' ), '#' ),
+				'id'      => 'enabled',
+				'type'    => 'radio_inline',
+				'options' => [
+					'enabled'  => __( 'Enabled', 'give' ),
+					'disabled' => __( 'Disabled', 'give' ),
+				],
+				'default' => 'enabled',
+			],
 			[
 				'id'         => 'headline',
 				'name'       => __( 'Headline', 'give' ),
 				'desc'       => __( 'Do you want to customize the headline for this form? We recommend keeping it to no more than 8 words as a best practive. If no title is provided the fallback will be your form’s post title.', 'give' ),
 				'type'       => 'text',
 				'attributes' => [
-					'placeholder' => __( 'Campaign Heading', 'give' ),
+					'placeholder' => $post->post_title ? $post->post_title : __( 'Campaign Heading', 'give' ),
```

Comment:
Instead of having duplicate code can we create a variable `$campaignHeading` and assign it to `placeholder` and `desc`

Reply:
Great idea! I've now stored dynamic defaults in variables at the top of the file, ref: https://github.com/impress-org/givewp/commit/7120fbfa883d6ea84ec6413d51b71ee474005d1d